<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAMs</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List of Video Cards</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetGPUs">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Product Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Memory Size</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Memory Speed</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
